scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,119 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script was written as part of a project to migrate
+# split repositories to a mono repository google-cloud-python.
+# This script will delete all files in a split repository and
+# update the README to indicate that the repository is archived.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+# Args:
+#   split-repo-name - The name of the split repository. For example `python-access-approval`.
+#   artifact-name - The name of the artifact on PyPI. For example `google-cloud-access-approval`.
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>"
+  exit 1
+fi
+
+# The name of the split repository. (e.g. python-access-approval)
+SPLIT_REPO=$1
+# The name of the artifact on PyPI. (e.g. google-cloud-access-approval)
+ARTIFACT_NAME=$2
+
+# Quick clean up in case a folder already exists in /tmp/
+rm -rf "/tmp/${SPLIT_REPO}"
+
+# Clone the split repo into /tmp/<name of split repo>
+git clone "git@github.com:googleapis/${SPLIT_REPO}.git" "/tmp/${SPLIT_REPO}"
+
+# cd into the split repo clone
+cd "/tmp/${SPLIT_REPO}"
+
+# Create a git branch
+git checkout -b 'delete-library'
+
+# Quick check to make sure there are no handwritten samples. 
+# Handwritten samples should be migrated to python-docs-samples before repository code is deleted.
+# Exclude the samples/generated_samples/* directory which contains autogenerated samples.
+NUM_PY_SAMPLES_FILES=$(find samples -type f -name "*.py" ! -path "samples/generated_samples/*" | wc -l)
+
+# Fail if there are handwritten samples.
+if [[ $NUM_PY_SAMPLES_FILES != "0" ]]; then
+  echo "Please migrate handwritten samples to python-docs-samples before deleting the source."
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to ensure that autogenerated files are not excluded.
+STANDARD_EXCLUDES=$(grep owlbot.py -e 'excludes=\["\*\*/gapic_version.py"]' | wc -l)
+
+if [[ $STANDARD_EXCLUDES != "1" ]]; then
+  echo "A non-standard value for `excludes` is not supported in the repository"
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific chek is to make sure that the owlbot.py file does not contain `s.replace` or `sed`
+# which has historically been used to modify the generated code.
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -e 's.replace' | wc -l)
+if [[ $NUM_STRING_REPLACEMENTS != "0" ]]; then
+  echo "s.replace() is not supported in the python mono repository"
+  exit 1
+fi
+
+NUM_STRING_REPLACEMENTS=$(grep owlbot.py -we 'sed' | wc -l)
```

Comment:
I haven't been able to reproduce this issue. 

Note: no occurrence of `sed` was found below because I'm using the `-w` option of `grep`
```
echo "Licensed" | grep -we 'sed' | grep -v "# Licensed"
```

Whereas, without the `-w` option there is 1 occurrence of `sed` in license
```
echo "Licensed" | grep -e 'sed' | grep -v "# Licensed"
```

From https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_04_02.html, `If you want to find a string that is a separate word (enclosed by spaces), it is better use the -w,`

Reply:
Oh, cool. I didn't notice the `-w` nor know what it does. My grepping otherwise did not use `-w`, so I ran into this issue.